switch/switch.js

```diff
@@ -0,0 +1,144 @@
+import {getStyle, getPseudoStyle} from './style-utils.js'
+
+class Switch extends HTMLInputElement
+{
+    _isDragging = false;
+    _recentlyDragged = false;
+    _thumbsize = 0;
+    _padding = 0;
+    _bounds = {
```

Comment:
```suggestion
    #isDragging = false;
    #recentlyDragged = false;
    #thumbsize = 0;
    #padding = 0;
    #bounds = {
```

don't need to mock private fields anymore, can have actual private class properties 👍🏻 
i tried it out on a local checkout, works great. you do need to update all the other `this._` references too (in case you accept this suggestion here in github)

Reply:
Cromium has support for these, I too would love to use em, but firefox is lagging behind

Reply:
tested locally on firefox 90, works great, and the caniuse shows support for all the majors https://caniuse.com/?search=private%20class

Reply:
oooh wow, finally, of course right after I am done migrating to typescript for my own project 🤣